data/filters/youtube-cleanup.yaml

```
@@ -56,6 +63,10 @@ tests:
       remove-comment-section: true
     output: |
       www.youtube.com###comments #contents:remove()
+  - params:
+      remove-comments-with-words: true
+    output: |
+      www.youtube.com##ytd-comments #content-text:has-text(/\b\b/i):upward(ytd-comment-thread-renderer)
```

Your template looks good, but you need to test with an actual word list, not `true`. This should work:
```suggestion
  - params:
      remove-comments-with-words:
        - word1
        - word pair2
    output: |
      www.youtube.com##ytd-comments #content-text:has-text(/\bword1\b/i):upward(ytd-comment-thread-renderer)
      www.youtube.com##ytd-comments #content-text:has-text(/\bword pair2\b/i):upward(ytd-comment-thread-renderer)
```